tests/examples/basic_features_test.js

```diff
@@ -1,20 +1,53 @@
 'use strict';
 
-const { suite, test, before, assert } = require('../../testy');
+const { suite, test, before, after, assert } = require('../../testy');
+const TestSuite = require('../../lib/test_suite');
+const { Asserter } = require('../../lib/asserter');
+const TestRunner = require('../../lib/test_runner');
+const { aPassingTest } = require('../support/tests_factory');
 
 suite('testing testy - basic features', () => {
   const circular = {}; circular.yourself = circular;
   let myVar = 8;
   
   before(() => myVar = 7);
+  after(() => myVar = undefined);
   
   test('tests with body', () => {
     const pepe = { nombre: 'pepe' };
     assert.that(pepe.nombre).isEqualTo('pepe');
   });
   
   test('before hook can be used', () => assert.areEqual(myVar, 7));
+  test('after hook can be used', () => {
```

Comment:
:thought_balloon: I agree this test can be moved to `test_suite_test.js` because we already have the context ready in that suite

Reply:
done